{{cookiecutter.hyphenated}}/src/{{cookiecutter.underscored}}/config.py

```diff
@@ -0,0 +1,29 @@
+"""API configuration settings.
+
+See: https://fastapi.tiangolo.com/advanced/settings/?h=conf#the-config-file
+"""
+from functools import lru_cache
+from pathlib import Path
+
+from pydantic import BaseSettings
+
+
+@lru_cache()
+def get_settings():
+    """Return the configuration settings.
+
+    Since lru_cache is used, the settings are read once from the file
+    and then returned from cache for all subsequent calls.
+    """
+    return Settings()
+
+
+class Settings(BaseSettings):
+    """Shared Application Settings."""
+
+    # Settings for Security Dependencies
+    secret_key: str = "09d25e094faa6ca2556c818166b7a9563b93f7099f6f0f4caa6cf63b88e8d3e7"
```

Comment:
What is this secret_key used for? Should it be here and hardcoded?

Reply:
It's an example used for signing JWTs, which is used for the `fake_users_db` in `dependencies.py`. It's necessary for the tests to work, but someone using the JWT implementation is supposed to swap out the `fake_users_db` and the `secret_key` used for demo purposes.

Reply:
It's the same secret used in the official docs https://fastapi.tiangolo.com/advanced/security/oauth2-scopes/ (which should be a sign that this value is not a good secret to use in production).